### DISCUSSION TITLE
Controls in flow temperature

### DISCUSSION THREAD

**Discussion by limkinZero** [gekkekoe/esphome-ecodan-hp#47] [Ideas]
Hi,

Is it possible to control the temperature in hot/cold temperature flow mode?

**Comment by gekkekoe** [comment]
the climate takes the current operating mode into account.
if the mode is in heat/cool flow, then the temperature set via the climate is the corresponding flow temp.

**Comment by limkinZero** [comment]
Ok, thanks. Pensaba I thought the typical numerical controls were missing.

**Comment by gekkekoe** [comment]
you can set a default heat/cool mode in the config. Click on the heat/cool icon in the climate, it will switch to those default modes. see info main page

**Comment by limkinZero** [comment]
Thanks, I checked it. Runs perfect.  The only things I miss compared to f1p
or broker development is , a pair of sensors to get error codes and control
dhw as a water heater component to control modes and the hot water
temperature.

Buts in general it's an amazing job. Thanks!

**Comment by gekkekoe** [comment]
you can control dhw via the dhw climate? heating mode is controlled via the selection of ECO/NORMAL ?
The error codes have not been implented yet, but I did reverse engineer them. I did not see too much usage for it yet.

**Reply by F1p** [reply]
> you can control dhw via the dhw climate? heating mode is controlled via the selection of ECO/NORMAL ? The error codes have not been implented yet, but I did reverse engineer them. I did not see too much usage for it yet.

My use case is an automation that if error code ever changes it sends a notification to my devices

**Reply by gekkekoe** [reply]
that is actually a good idea :P will add them if I can find some spare time.
